apps/nextra/pages/en/build/create-aptos-dapp/templates/_meta.tsx

```diff
@@ -1,20 +1,32 @@
 export default {
   boilerplate: {
     title: "Boilerplate Template",
+    href: "https://learn.aptoslabs.com/en/dapp-templates/boilerplate-template",
```

Comment:
If there's no content here I think it's probably better to remove this from the sidebar and just add redirects